gis/school_safety_zones/schools.py

```diff
@@ -153,7 +154,7 @@ def is_int(n):
 
 #--------------------------------------------------------------------------------------------------
 
-def pull_from_sheet(con, service, year, spreadsheet, *args):
+def pull_from_sheet(con, service, year, spreadsheet, **kwargs):
```

Comment:
Is this function actually receiving keyword arguments?

Reply:
yes, it's used to send some variables over XCom to the slack alert function